tirocks/src/env/tests.rs

```diff
@@ -0,0 +1,99 @@
+use super::*;
```

Comment:
Is there a particular reason to move tests here?

Reply:
Just not to make env.rs too long. Besides this also has a minor effect that update tests won't invalidate common build caches.